src/main/java/com/speaker/service/MessageServiceImpl.java

```diff
@@ -22,22 +25,31 @@ public class MessageServiceImpl implements MessageService {
     private final MessageRepository messageRepository;
     private final AccountRepository accountRepository;
     private final MessageConvertor messageConvertor;
+    private final FieldValidators<MessageDTO> messageDTOValidators;
 
     @Override
     public List<Message> getMessagesByAccountId(int accountId) {
         return messageRepository.findAllByAccountId(accountId);
     }
 
-    @Override
-    public Response addMessage(List<MessageDTO> messageDTOs) {
+        @Override
+    public List<ValidatorError> addMessage(List<MessageDTO> messageDTOs) {
+        List<ValidatorError> validatorErrors = messageDTOs.stream()
```

Comment:
let's do two methods in `FieldValidators`:

1)  `List<ValidatorError> validate(T entity);`
2)  `List<ValidatorError> validate(List<T> enties);`

Reply:
Comments accepted, corrections made. The interface is extended